src/autoscaler/Makefile

```diff
@@ -28,6 +28,7 @@ EXTENSION_FILE := $(shell mktemp)
 
 export GOWORK=off
 BUILDFLAGS := -ldflags '-linkmode=external'
+export GOFIPS140=v1.0.0
```

Comment:
This is a bit of duplication, but required for the BOSH release. Once we stop building a BOSH release this could be removed.